packages/core-components/src/layout/Header/Header.tsx

```diff
@@ -70,6 +71,10 @@ const useStyles = makeStyles(
       wordBreak: 'break-word',
       fontSize: theme.typography.h3.fontSize,
       marginBottom: 0,
+      '& > div > span > span': {
+        display: 'unset',
+        // to fix title overflow issue
+      },
```

Comment:
This needs to be applied more directly to the element, the selector is drilling a bit too deep